docs/design/fabric-v2+/fpc-management.md

```diff
@@ -76,28 +76,33 @@ Users must aware that each FPC chaincode runs in a single enclave at a designate
 
 ## FPC Endorsement Policies
 
-The initial version of FPC restricts the set of allowed endorsement policies and currently
-supports a designated endorser policy only. That is, only a single endorsing peer is
-responsible to execute transactions of an FPC chaincode for the consortium.  As an FPC
-chaincode is protected by the use of Trusted Execution Technology, from a security
-perspective, the integrity guarantees gained from the endorsement model are similar to the guarantee provided in a typical Fabric system with multiple Peer endorsements.
-However, the designated endorser policy does not provide resilience in terms of availability and limits a particular chaincode's scalability.
-In future releases of FPC, we will address this limitation and relax this restriction by
-providing support for rich endorsement policies.
-
-To achieve the designated-peer endorsement policy, the consortium initially selects an organization (e.g., SampleOrg)
-that will host the designated endorsing peer.  The corresponding endorsement policy
-needs to be set to `OR('SampleOrg.peer')` and approved by the consortium using normal chaincode lifecycle agreement.
-Other expressions like `AND` or `OR` with multiple organizations are currently not allowed and
-may lead to unexpected behavior.
-
-Note that in order to use `SampleORG.peer`, `NodeOUs.Enable` must be set to `true`, otherwise
-`OR('SampleOrg.member')` must be used.
-
-For example:
-
-    peer lifecycle chaincode approveformyorg --channelID mychannel --name myfpc --signature-policy "OR('SampleOrg.peer')" ...
-
-An admin of SampleOrg is then responsible to invoke `lifecycle chaincode initEnclave` at
-a single peer and thereby determine the designated endorser for the `myfpc` chaincode.  Clients
-of the consortium have to make sure that they invoke transactions only at the designed endorser.
+In FPC, signatures by enclaves are the base of endorsements.
```

Comment:
I think we lose the reader here with "endorsement", and Dave was similarly confused.
Should we specify immediately the two-step procedure, endorsement+validation?

Reply:
Not sure why at this point there would be confusion? And to understand what the two-step procedure is, you have to first understand what endorsement really means in our context?  The only thing i could see is to swap the  rest of the paragraph with the paragraph below referencing the externalized endorsement.  But to me the current order seems more natural coming conceptually top-down to the low-level realization.

Reply:
It's a more general problem related to our use of "endorsement", which has a different meaning in FPC. Anyway, let's clarify that when needed.